news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,211 @@
+
+Thousands of users have Stefan to thank for their regular bundle of fresh maps to play, and we're extremely thankful for his incredible dedication and persistence.
+
+### Naxess
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Naxess.jpg)](https://osu.ppy.sh/users/8129817)
+
+On the subject of unsung heroes, Naxess' has pioneered the development of a suite of tools that have essentially come to shape the way the Beatmap Nominators and the Quality Assurance Team handles the constant influx of new beatmaps and things to check.
+
+Once Aiess and now Victoria, Naxess' creation stands guard in many a Discord server to guide masses of modders as they navigate through the challenges that come with ranking beatmaps, providing a service that hundreds of people now take largely for granted.
+
+His efforts have not gone unnoticed.
+
+### Kurokami
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Kurokami.jpg)](https://osu.ppy.sh/users/260933)
+
+A long term member of the Beatmap Nominators, then the Quality Assurance Team, Kurokami has been a regular presence in beatmapping-related affairs for literal years. His most notable contribution is his leadership of the Beatmap Spotlights project, and its continued effort to bring only the highest quality of beatmaps to bear for the community's appraisal.
+
+Through thick and thin, Kurokami's ceaseless direction of the project has kept it alive where circumstance would have doomed it to be forgotten, and it is this sheer determination and drive to see something done that we are tremendously grateful to benefit from.
+
+### p3n
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/p3n.jpg)](https://osu.ppy.sh/users/123703)
+
+Our initial reaction to p3n's name being raised was universally: "how does he not have this badge already?"
+
+So vast and numerous are p3n's contributions over the years - from creating a major metric used by the Quality Assurance Team to weigh and compare the activity of a modder in a given period of time, to entire written drafts and improvements on the basic modding/ranking system, his previous and enduring leadership of both the Beatmap Approval Team and the current QAT, assistance with the osu! World Cups and so, so much more.
+
+### shARPII
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/shARPII.jpg)](https://osu.ppy.sh/users/776257)
+
+Departed from the osu!team earlier in the year, shARPII was once the primary manager of the Global Moderation Team. Not merely content with one herculean task, shARPII also decided he'd write a utility to help the osu! World Cup staff handle the arduous process of handling multiplayer games from a referee's perspective, coding the program who is now affectionately dubbed *Script-chan* by the tournaments crew.
```

Comment:
shARPII retired in 2018. Also, in the last part, `who` -> `which` is more correct imo but I think this is pretty minor

Reply:
fixed